docs/user-guide/configure.md

```diff
@@ -234,6 +234,16 @@ To use one, add a `"plugins"` array to your config, containing "locaters" identi
 - npm module name
 - absolute path
 - path relative to the invoking configuration file
+- stylelint plugin object of format
+
+```js
+{
+  ruleName: "plugin-name",
+  rule: function plugin(opts) { /* Plugin Code */ }
+}
+```
+
+> Read more about writing [Stylint Plugins here](../developer-guide/plugins.md).
```

Comment:
I also agree with @jeddy3's suggestion. Furthermore, **"plugin objects"** may be more accurate than "plugin functions" because a Stylelint plugin is an object:

https://github.com/stylelint/stylelint/blob/2290f557cb028242d978b06465dc108aa04a0c3a/lib/createPlugin.js#L6-L11